scale-tests/batch_test.py

```diff
@@ -123,10 +128,28 @@ def submit_loop(submits_per_min: int, dispatchers: typing.List[typing.Dict], use
                  "--conf", "spark.mesos.driver.failoverTimeout={}".format(args["--spark-mesos-driver-failover-timeout"])
                 ]
 
+    if args["--spark-mesos-executor-gpus"]:
+        MEMORY_MULTIPLIER = 20
```

Comment:
Would it make sense to also move the construction of these config options to a helper function? e.g. `get_gpu_conf_args`?